web/src/styles/legacy.css

```diff
@@ -32,8 +32,11 @@
     'avenir next', avenir, 'helvetica neue', helvetica, roboto, noto, 'segoe ui',
     arial, sans-serif;
 
-  --font-size-xl: 1.25rem; /* 20px */
-  --line-height-xl: 1.875rem; /* 30px */
+  --font-size-xl: 1.5rem; /* these need to be properly assigned by Viz Dee */
+  --line-height-xl: 2rem;
```

Comment:
@jenniferthibault These are just stubbed in values to get the state title in the sidebar closer to the size reflected in your Sketch screenshot. Let me know if there are real values you'd like me to plug in!

Reply:
Added what I'm using in the design files (which are the settings for an [H3](https://design.cms.gov/style/typography/) in the new CMS design system). I'm not sure if these will be future-proof if the `text-xl` class is also from that system, but should work ok for now?